packages/attestation-service/package.json

```diff
@@ -19,6 +19,7 @@
     "prepublishOnly": "yarn build:gen && yarn build",
     "start-ts": "TS_NODE_FILES=true ts-node src/index.ts",
     "start": "node lib/index.js",
+    "db:create": "mkdir -p db && touch db/dev.db",
```

Comment:
I don't think this should be here? I.e. it should not be the concern of this application to create databases other than for development and testing

Reply:
that command was to make it consistent with the documentation found in the README.md of the attestation-service and the comment made about not including the `dev` suffix.

What do you suggest? Add the command or keep the `yarn run db:create:dev` instruction in the README?